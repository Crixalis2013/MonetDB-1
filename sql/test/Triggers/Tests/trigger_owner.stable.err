stderr of test 'trigger_owner` in directory 'sql/test/Triggers` itself:


# 14:14:17 >  
# 14:14:17 >  Mtimeout -timeout 180 Mserver "--config=/ufs/goncalve/scratch/MonetDB/monetdb4/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/ufs/goncalve/scratch/MonetDB/monetdb4/lib/MonetDB:/ufs/goncalve/scratch/MonetDB/monetdb4/lib/MonetDB/lib:/ufs/goncalve/scratch/MonetDB/monetdb4/lib/MonetDB/bin" --set "gdk_dbfarm=/ufs/goncalve/scratch/MonetDB/monetdb4/var/MonetDB/dbfarm" --set "sql_logdir=/ufs/goncalve/scratch/MonetDB/monetdb4/var/MonetDB/sql_logs" --set "xquery_logdir=something" --set mapi_port=35358 --set monet_prompt= --trace "--dbname=mTests_src_test_Triggers" --dbinit=" module(sql_server);" ; echo ; echo Over..
# 14:14:17 >  


# 14:14:18 >  
# 14:14:18 >  Done.
# 14:14:18 >  

