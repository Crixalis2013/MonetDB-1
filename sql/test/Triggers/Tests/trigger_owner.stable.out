stdout of test 'trigger_owner` in directory 'sql/test/Triggers` itself:


# 20:28:20 >  
# 20:28:20 >   mserver5 "--config=/ufs/niels/scratch/monetdb/Linux-x86_64/etc/monetdb5.conf" --debug=10 --set gdk_nr_threads=4 --set "monet_mod_path=/ufs/niels/scratch/monetdb/Linux-x86_64/lib/MonetDB5:/ufs/niels/scratch/monetdb/Linux-x86_64/lib/MonetDB5/lib:/ufs/niels/scratch/monetdb/Linux-x86_64/lib/MonetDB5/bin" --set "gdk_dbfarm=/ufs/niels/scratch/monetdb/Linux-x86_64/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/niels/scratch/monetdb/Linux-x86_64/var/MonetDB5/sql_logs"  --set mapi_open=true --set xrpc_open=true --set mapi_port=35138 --set xrpc_port=49382 --set monet_prompt= --trace  "--dbname=mTests_src_test_Triggers" --set mal_listing=0 "--dbinit= include sql;" ; echo ; echo Over..
# 20:28:20 >  


# 14:14:18 >  
# 14:14:18 >  Done.
# 14:14:18 >  

